### PR TITLE
nodejs 20.11.1

### PR DESCRIPTION
**Pull request by pheiduck**
# Proposed Changes

- Bump to nodejs 20.11.1

## Checklist

- [x] Tested locally
- [x] Ran `yarn ci` to test my code
- [x] Did not add any unnecessary changes
- [ ] All my changes appear after other changes in each file
- [ ] Included a screenshot (if adding a new button)
- [ ] 🚀

### CODE REVIEW

**Comment by timothystewart6** [comment]
Hey, thanks!  I think this needs to be bumped in a few places!  Just do a find all for that version!

File: yarn.lock
Comment (timothystewart6) [inline yarn.lock:1]: Actually, we can't change it in the lock, we need to change it in `package.json` and then run `yarn` (if there is an updated version).

Comment (timothystewart6) [reply yarn.lock:1]: Also, there are still quite a few places the node version is set, `.nvmrc`,  github actions, etc...

Comment (pheiduck) [reply yarn.lock:1]: So I do have to revert the changes from the varn.lock file?

Comment (timothystewart6) [reply yarn.lock:1]: Yes please, the lock is generated automatically when `package.json` changes.  It pins exact version of npm modules after running `yarn install`

**Comment by timothystewart6** [comment]
Thank you!

**Pull request merged**